### PR TITLE
♻️ Simplify assertEq for arrays

### PR DESCRIPTION
**Pull request by mds1**
@ZeroEkkusu likely won't be able to get to the tests today so if wanted to fix those that would be appreciated!

cc @paulrberg

### CODE REVIEW

File: src/Test.sol [hunk 1]
```diff
@@ -226,58 +230,32 @@ abstract contract Test is DSTest, Script {
     }
 
     function assertEq(uint256[] memory a, uint256[] memory b) internal {
-        if (a.length != b.length) {
-            emit log("Error: a.length == b.length not satisfied");
+        if (keccak256(abi.encode(a)) != keccak256(abi.encode(b))) {
+            emit log("Error: a == b not satisfied [uint[]]");
+            emit log_named_array("  Expected", b);
+            emit log_named_array("    Actual", a);
```
Comment (ZeroEkkusu) [inline src/Test.sol:236]: Forge doesn't support printing these, does it? @mds1 

They show up only in traces.

**Review by ZeroEkkusu — APPROVED** [review]
Ready to merge once we resolve the convo above.

File: src/Test.sol (hunk 1 above)
Comment (mds1) [reply src/Test.sol:236]: Thanks for handling the tests!

Good point, it doesn't currently, but it shouldn't be too hard to support so we can create an issue upstream to track

**Comment by paulrberg** [comment]
This is awesome guys! Just found the time to review the code changes, and it LGTM.

**Pull request merged**